### ISSUE TITLE
ERROR:EDK - IPNAME

### ISSUE THREAD

**Issue by eselahe** [cmlab/NetFPGA-1G-CML#8]
Hi everyone,
I have some problems when I want to simulate and implement some reference projects. I expressed them in issues #5 reference_nic_nf_cml test and  #6 Java GUI make. These errors are about EDK - IPNAME. Also in the issue #2 loopback test, it has been said these errors occurred. 

please help me for solving these problems.
Thanks in advance!

**Comment by arashkh** [comment]
I'm having the same problem when I try to make any of the individual projects.
I guess EDK does not know where to look for the MDP file of the pcores. There must be a way to let it know of the location.

**Comment by jhirata** [comment]
eselahe,

I've talked with Arash and I've resolved his problem. It was not necessarily related to your problem though. In regards to your issue, all of the posts on the web that I've seen indicate that it might be an issue with the Xilinx tool installation. Have you been able to make any progress on your issue? 

Thanks,
Jay

**Comment by eselahe** [comment]
Thanks for your comment. Unfortunately, I don't know what exactly I should do.

**Comment by jhirata** [comment]
Have you tried building any of the 10G projects? Have you had any success building any other projects unrelated to the NetFPGA? Have you tried building a Microblaze system using EDK on your own?

**Comment by eselahe** [comment]
Hi jay,
I could implement reference_nic_nf_cml and reference_router_nf1_cml projects on NetFPGA and I have programmed my NetFPGA with these projects, successfully. For example when I use NetFPGA as NIC and I connect NetFPGA Ethernet ports to other PC, they can ping each other.
But I can't run the tests inside the reference_nic_nf_cml project, successfully. I get some errors about EDK - IPNAME which I expressed them in issues #5 reference_nic_nf_cml test.
Also about reference_router_nf1_cml project, my board is programmed, successfully, but the register infrastructure  for this project didn't built by entering the "hw" directory in the design and running "make regs" and I get errors about EDK - IPNAME agin which I expressed them in issues #6 Java GUI make.
I only have tried building these projects.I can't understand second your question and about third question, no, I haven't tried building a Microblaze system using EDK on my own.

Thanks,
elahe
